cluster/examples/kubernetes/ceph/upgrade-from-v0.8-replace.yaml

```diff
@@ -45,7 +45,6 @@ rules:
   resources:
   - deployments
   - daemonsets
-  - replicasets
```

Comment:
no need to update this v0.8 upgrade file, right?

Reply:
That should be removed when I refactor with master.